.ci/test-spelling-unknown-words.sh

```diff
@@ -42,7 +42,7 @@ echo "Clean up from previous run"
 rm -f "$run_output"
 
 echo "Run w"
-(git 'ls-files' -z 2> /dev/null || hg locate -0) |\
+(git ls-files 'src/test/' '*.java' '*.txt' '*.xml' '*.html' '*.properties' -z 2> /dev/null) |\
```

Comment:
revert this line.
it should be only on your local to help you see weird names in test Inputs.

There should be updates in InputXxxxxxxx files to fix bad named to good words.